src/app/file-manager/services/firecloud-mock.service.ts

```diff
@@ -0,0 +1,34 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { FirecloudService } from './firecloud.service'
+
+@Injectable()
+export class FirecloudMockService {
+
+
+  constructor(private http: HttpClient, private firecloudService: FirecloudService) { }
+
+
+  public getUserFirecloudWorkspaces(optional: boolean) {
+      return this.http.get<any>('assets/demo/workspaces.json')
+      .map((resp: any) => {
+        let workspaces: any[] = new Array();
+        resp.forEach(item => {
+          let workspace = this.firecloudService.getWorkspaceData(item, optional);
+          if(workspace !== null){
```

Comment:
I think the default formatting rules specify a space between `if` and `{`. If you re-indent your code in an IDE like intellij you should get the correct behavior.

Reply:
Also this can be accomplished with the TSLint extension for VS Code.